model/otel/metrics.yaml

```diff
@@ -8,7 +8,8 @@ groups:
     unit: "{span}"
     note: |
       For spans with `recording=true`: Implementations MUST record both `otel.sdk.span.live` and `otel.sdk.span.ended`.
-      For spans with `recording=false`: If implementations decide to record this metric, they MUST also record `otel.sdk.span.ended`.
+      For spans with `recording=false`: Implementations MAY omit recording `otel.sdk.span.live` (e.g. for performance or simplicity reasons).
```

Comment:
```suggestion
      Spans with `recording=false` are not considered live for purposes of calculating `otel.sdk.span.live`.
```

Reply:
I would prefer having a consistent meaning of live, instead of making it optional with a MAY.  I would also consider updating the description to say "The number of created, **recording** spans for which the end..."